mmv1/third_party/terraform/services/container/resource_container_node_pool_test.go.tmpl

```diff
@@ -933,6 +933,35 @@ func TestAccContainerNodePool_withSandboxConfig(t *testing.T) {
 }
 {{- end }}
 
+func TestAccContainerNodePool_withSandboxConfigType(t *testing.T) {
+	t.Parallel()
+
+	cluster := fmt.Sprintf("tf-test-cluster-%s", acctest.RandString(t, 10))
+	np := fmt.Sprintf("tf-test-np-%s", acctest.RandString(t, 10))
+	networkName := acctest.BootstrapSharedTestNetwork(t, "gke-cluster")
+	subnetworkName := acctest.BootstrapSubnet(t, "gke-cluster", networkName)
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerNodePool_withSandboxConfigType(cluster, np, networkName, subnetworkName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_container_node_pool.with_sandbox_config",
+						"node_config.0.sandbox_config.0.type", "GVISOR"),
+				),
```

Comment:
This check is failing with the error:

```
resource_container_node_pool_test.go:954: Step 1/2 error: Check failed: Check 1/2 error: Check 1/1 error: google_container_node_pool.with_sandbox_config: Attribute 'node_config.0.sandbox_config.0.type' not found
```

I'd suggest just removing the check; importstateverify should verify this.

```suggestion
```